app/controllers/team/projects_controller.rb

```diff
@@ -4,33 +4,17 @@ class Team::ProjectsController < ApplicationController
 
   def in_progress
     authorize Project, :index?
-    @pager, @projects = pagy(Project.assigned_to_regional_caseworker_team.in_progress.includes(:assigned_to))
-
-    pre_fetch_establishments(@projects)
```

Comment:
Loosing the pre fetching will make these views really slow. I would add the pre fetchers to the service, you could have method that fetches and returns the projects, something like:

```
private def pre_fetch_establishments_and_trusts(projects)
   EstablishmentsFetcherService.new(projects).call!
  TrustsFetcherService.new(projects).call!
  projects
end
```

Reply:
Ah yes I saw you'd done that on the ConversionProjectsFetcherService 👍